### ISSUE TITLE
Labels for the de Broglie combo box

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/models-of-the-hydrogen-atom#36]
Related to #1 design refresh ...

The de Broglie model has 3 representations, selectable via a combo box:

<img width="380" alt="screenshot_1756" src="https://user-images.githubusercontent.com/3046552/177401928-b84d897a-5be4-4ab5-bb7b-d1a0b37fc268.png">

Each of those views is a different **mapping of amplitude** to some visual characteristic, as follows:

* Radial view maps amplitude to radial distance in 2D
* 3D view maps amplitude to height in 3D
* Brightness view maps amplitude to brightness in 2D

I think we should reconsider the labels that we're using in the combo box.  "radial view" is lousy - "distance" is the important part, "radial" is the adjective that tells you what distance. "3D view" is the worst; the fact that it's 3D is the least important part of the description.  I'm also not a fan of the word "view" appearing in every label.

Here's a strawman proposal:
* Radial Distance
* 3D Height
* Brightness

Or maybe the labels (or a label for the combo box?) need to indicate that this is a mapping of amplitude (and amplitude of what?)

@arouinfar thoughts?

**Comment by pixelzoom** [comment]
@ariel-phet suggested replacing the combobox with 3 radio buttons, and using icons instead of text.

**Comment by arouinfar** [comment]
Here are a set of iconic radio buttons 
<img width="259" alt="image" src="https://user-images.githubusercontent.com/8419308/180280272-565dbd4d-959d-41f5-8041-d4d32b9dfebd.png">

And here they are in context (screenshot uses Bohr model so the deBroglie ComboBox wasn't in the way)
<img width="743" alt="image" src="https://user-images.githubusercontent.com/8419308/180280123-ef128b6e-6b09-4145-b16b-a53685bf577a.png">

**Comment by pixelzoom** [comment]
@arouinfar where can I find these icons and there asset file(s)?

**Comment by arouinfar** [comment]
Unassigning myself until work resumes on this sim.

**Comment by pixelzoom** [comment]
Like the "white" and "monochromatic" radio button labels for the light, I'm wondering if it's important to have words shown in the UI for these representations.   

I still prefer these labels to the icons:
* Radial Distance
* 3D Height
* Brightness

**Comment by pixelzoom** [comment]
8/13/24 design meeting, @DianaTavares @arouinfar @Nancy-Salpepi @pixelzoom 

We decided to stick with the combo box with text items for these reasons:
* Text provides a way for teachers and others to refer to the different representation choices.
* Text provides common terminology to be used when describing the mappings of amplitude.
* Icons would be expensive to create programmatically. Creating them as artwork would bake in the colors, and not work with Color Editor.

In the above commits, I changed the strings and internal code to use these terms:

* Radial Distance
* 3D Height
* Brightness

I'll close this issue. We can reopen if we change our minds.
